src/pages/signin/PasswordForm.js

```diff
@@ -57,12 +57,19 @@ class PasswordForm extends React.Component {
      */
     validateAndSubmitForm() {
         if (!this.state.password.trim()
-            || (this.props.account.requiresTwoFactorAuth && !this.state.twoFactorAuthCode.trim())
+            && (this.props.account.requiresTwoFactorAuth && !this.state.twoFactorAuthCode.trim())
```

Comment:
Doesn't change the result, but the parenthesis around `this.props.account.requiresTwoFactorAuth && !this.state.twoFactorAuthCode.trim()` aren't necessary.

Reply:
Removed the unnecessary brackets and also moved the conditions into single line.

Reply:
Awesome, thanks!